databases/relational-databases.md

```diff
@@ -31,6 +33,43 @@ To understand, query, and insert information into a relational database, a techn
 
 ### Specific Things To Teach
 
+- The specific problems a database solves
+- Users of databases
+- Structure of a relational database; tables with columns and references to other tables
+- Few basic column types
+- The four basic SQL verbs and their use
+- Bonus topic for the advanced: foreign key constraints
+- Explain different databases with slightly different dialects (MySQL - PostgreSQL)
+- Explain existence of other types of non-relational, non-sql databases that will be encountered
+
+### Materials
+
+#### Education
+
+- [Khan Academy's introductory SQL, videos](https://www.khanacademy.org/computing/computer-programming/sql/sql-basics/v/welcome-to-sql)
+- [Difference between MySQL and PostgreSQL](https://techdifferences.com/difference-between-mysql-and-postgresql.htmls)
+- [LucidChart: What is a Database Model?](https://www.lucidchart.com/pages/database-diagram/database-models)
+- [Structure of a relational database](https://www.lucidchart.com/pages/database-diagram/database-design)
+- [Relational Model](https://www.lucidchart.com/pages/database-diagram/database-models?a=1)
+- [four basic SQL verbs](https://blog.sqlauthority.com/2008/05/13/sql-server-four-basic-sql-statements-sql-operations/)
+- [streaming(cassandra)](https://dzone.com/articles/cassandra-sink-for-spark-structured-streaming)
+- [Describing an ORM](https://www.techopedia.com/definition/24200/object-relational-mapping--orm):
+
+
+
+
+#### Tools
+
+- [SQL Fiddle](http://sqlfiddle.com)
+
+### Lesson
+
+- [Slides](https://drive.google.com/open?id=1xK7_t_yJcu4RcBkj0Gv-t5uyBCNr0g4cHKqAJSxNwY0) | [Video Walkthrough of Slides](https://drive.google.com/file/d/1V0bk3fH_8PsRE3Vz4J3qe3TTiqBClT6y/view)
+
+Also briefly review: [LucidChart: What is a Database Model?](https://www.lucidchart.com/pages/database-diagram/database-models). Be sure to read the [Relational Model](https://www.lucidchart.com/pages/database-diagram/database-models?a=1) section, and briefly skim over the rest just so you're aware that there are many types of database models. The only one we're working with here is the relational model, so don't worry about learning the rest yet. 
+
+- Go through the complete tutorial mentioned in the Materials Section. [SQL Fiddle](http://sqlfiddle.com) is the tool where you can implement your sql knowledge. Practically implement creating a database schema and perform operations on it using the tool.
```

Comment:
Which tutorial do they need to complete? Is sql fiddle a tutorial

Reply:
No its a tool. I have mentioned in the description.

Reply:
Then which tutorial do they need to complete?

Reply:
The one that are mentioned in material section

Reply:
None of them say tutorial. Please just tell me which one, or change the sentence yourself.

Reply:
Are we having translation problems here?

Reply:
Do you mean” go through every material in the Materials section” ?

Reply:
Yes.

Reply:
I see, thank you.